app/Http/Controllers/Three/SignupsController.php

```diff
@@ -48,7 +48,6 @@ public function __construct(SignupService $signups)
     public function store(Request $request)
     {
         $this->validate($request, [
-            'northstar_id' => 'required|string',
             'campaign_id' => 'required',
             'campaign_run_id' => 'int',
             'why_participated' => 'string',
```

Comment:
can we delete `source` (below) since we're now storing this using `token()->client()` in the Signup Repository?

Reply:
Yep!